beacon-chain/blockchain/chain_info.go

```diff
@@ -68,12 +67,6 @@ type FinalizationFetcher interface {
 	PreviousJustifiedCheckpt() *ethpb.Checkpoint
 }
 
-// ParticipationFetcher defines a common interface for methods in blockchain service which
-// directly retrieves validator participation related data.
-type ParticipationFetcher interface {
```

Comment:
Why was all of this removed?

Reply:
It was unused, see notes in PR description.